src/components/_forms.css

```diff
@@ -60,7 +60,7 @@
   appearance: none;
   background: transparent no-repeat right 1rem center url(https://envoy.com/images/icons/select-carat.svg);
   font-family: inherit;
-  font-size: var(--h6);
+  font-size: var(--h5);
```

Comment:
Looks good. Can you also incorporate the changes to border-radius that we discussed in the channel? Buttons and form elements (inputs, selects, textareas) should have a 4px border radius instead of the 6px we use for everything else.